knownprojects_build/_sql/combine.sql

```diff
@@ -71,24 +124,32 @@ _dcp_planneradded as (
         portion_bu::numeric as portion_built_by_2025,
         portion__1::numeric as portion_built_by_2035,
         portion__2::numeric as portion_built_by_2055,
-        wkb_geometry::geometry as geom
-    FROM dcp_planneradded
+        wkb_geometry::geometry as geom,
+        flag_nycha(a::text) as nycha,
+        flag_gq(a::text) as gq,
+        flag_senior_housing(a::text) as senior_housing,
+        flag_assisted_living(a::text) as assisted_living
+    FROM dcp_planneradded a
 ),
 _dcp_n_study_future as (
     SELECT
         'Future Neighborhood Studies' as source, 
         a.uid as record_id,
         array_agg(a.uid) as record_id_input,
```

Comment:
Does it make sense to have the array_agg here if a.uid is in the group by statement?